### DISCUSSION TITLE
Graph: Add quick filter input field for Legend values in widget

### DISCUSSION THREAD

**Discussion by MurzNN** [grafana/grafana#38213] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Will be good to add small input field to type keywords for quick filter-out the whole list of Legend values by containing it's keyword. It can be located in legend name header of Table style, where now is empty place (when enabled Values options).

And maybe similar filter for non-table lists of values. For not waste space in regular charts with only several Legend values, we can show the filter input on-demand, only if chart have scroll line in legend area (or maybe on hover).

**Why is this needed**:
Some charts have very long list of Legend values, and it is too hard to find needed ones. Via quick filter we will can filter-out them by some keyword, here is example of problem and proposed solution:
![image](https://user-images.githubusercontent.com/336662/106111181-ca121400-615c-11eb-8f55-9407f51ceaa8.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by nicolas-san** [comment]
Hi, any news on this one ?
I was searching the same feature :)

**Comment by ajay-sainy** [comment]
I was looking for the same, created chrome/edge extension - https://github.com/ajay-sainy/GrafanaLegendFilter

**Comment by smnsivaprakash** [comment]
I'm also looking for this requirement. Do you have any updates on this so far?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83021. Feel free to continue the discussion around this there. Thank you!

**Comment by MurzNN** [comment]
Posted as an issue here https://github.com/grafana/grafana/issues/83043

**Comment by gelicia** [comment]
@MurzNN My apologies for this not being clear - I migrated this already for you. 

The issue is https://github.com/grafana/grafana/issues/83021
